research/src/pages/tabs.research.parts.mdx

```diff
@@ -0,0 +1,439 @@
+Oragami | Not specifically supported, but markup based and seems to support embedding custom content
+Semantic UI | Not explicitly supported ('menus' separate from the component they call 'tabs' which is more of a set of mutually exclusive selectable sections, but looks like these menus can contain embedded custom content)
+Shoelace | (not specifically supported, but seems to support embedding custom content)
+Vaadin | (not specifically supported, but explicitly supports embedding custom content)
+
+
+
+<figure>
+  <img src="https://lh4.googleusercontent.com/JMQRH-7jzLBgAnLtOr7zTNKL1Oe0ZKgV71wkwen1MRrXnHi1BNSDCAo_ZW0cGyWiVkus3eEkDGs2BQW05pBG5Ggp7rDLQzcSZFfhBx4xten5DsE-ZYJmIvF_6pFSkLVU-Mz34PB6" />
+  <figcaption>
+Some systems which support icons allow for tabs that contain only icons, others do not.   Material is an example of the former. Note that in these cases, it is improtant that non-visible text alternatives must be provided for a11y, though most systems don't specifically require it.
+    </figcaption>
+</figure>
+
+
+Most design systems recommend homogeneity in the same Tab Bar (that is, against mixing modes of combination of text/icon). Some actively enforce this, via API, some don't.
+
+
+#### Directionality/Mirroring
+There are some well established expectations with regard to internationalization for certain parts of Tab Sets.  In right-to-left languages, for example, generally speaking some kind of RTL mirroring applies - however, it will work work differently depending on a number of other factors.  Some of these are very well established, others less so.  For example, when tabs are grouped in the in the horizontal top position, in RTL the order of the tabs is reversed, and the icon/text flip too.  Keyboard controls, however, continue to maintain spatial order (that is, pressing 'left arrow' will move selection to the tab displayed to the left of the currently selected and so on. First and last shortcuts will be impacted as per the directionality.  As systems add additional elements or orientations this becomes slightly less clear.
+
+If the component offers some additional afforandance for overflow (see overflow), for example, this may or may not have effects (paging or next/back buttons would work spatially, while a dropdown should appear to the left but options would be in ...document order?).
+
+
+<figure>
+  <img src="https://lh4.googleusercontent.com/SZMUPImNdUKl-TL7z_RBsNIqj2tcofFM2IjGV2qa4025hsAPylWVeAXC04SmeOc2JwIT-6kimj_VObEhJ4YSxSA3OjxYZkHpaTsFP97izDxwW_Y0pwfC_EAo9CBSvXj2QbZQ4yhS" />
+  <figcaption>Material design with icon and text in horizontal top position in LTR, and the same in RTL</figcaption>
+
+</figure>
+
+Mirroring should work similarly in horizontal bottom positioning.  However, if left and right positioned tabs are supported, they will reverse the order of their alignment and their icon/text relationship, but not the order of the tabs themselves.
+
+
+##### Icon Alignment 
```

Comment:
Should this be a `####` heading? It seems small compared to Directionality/Mirroring heading and it is hard to tell it's a subheading. If this is a subsection of Directionality/Mirroring then this may be another example of @davatron5000 recommending some styling adjustments to aid in these larger research documents.

Reply:
Yes it is a subheading/subsection - I agree with @davatron5000 that this could use style help